### PR TITLE
[lldb] Add implicit modules to the list of hand loaded modules

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test macOS

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -8307,13 +8307,33 @@ bool SwiftASTContext::GetImplicitImports(
...
-                                                 ProcessSP process) {
+void SwiftASTContext::LoadImplicitModules(TargetSP target, ProcessSP process,
+                                          ExecutionContextScope &exe_scope) {
   auto load_module = [&](ConstString module_name) {
     SourceModule module_info;
     module_info.path.push_back(module_name);
     Status err;
     LoadOneModule(module_info, *this, process, true, err);
+    if (err.Fail()) {
+      LOG_PRINTF(GetLog(LLDBLog::Types),
```
Comment (adrian-prantl) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:8318]: FWIW, I think these should be HEALTH logs

**Review by adrian-prantl — APPROVED** [review]

**Pull request merged**